pkg/aws/v1alpha3/cluster/validate_cluster.go

```diff
@@ -119,6 +143,75 @@ func (v *Validator) ValidateUpdate(request *admissionv1.AdmissionRequest) (bool,
 	return true, nil
 }
 
+func (v *Validator) ClusterAnnotationUpgradeTimeIsValid(cluster *capiv1alpha3.Cluster) error {
+	if updateTime, ok := cluster.GetAnnotations()[annotation.UpdateScheduleTargetTime]; ok {
+		if !UpgradeScheduleTimeIsValid(updateTime) {
+			return microerror.Maskf(notAllowedError,
+				fmt.Sprintf("Cluster annotation '%s' value '%s' is not valid. Value must be in RFC822 format and should be a date in the future not more than 6 months away.",
+					annotation.UpdateScheduleTargetTime,
+					updateTime),
+			)
+		}
+	}
+	return nil
+}
+
+func UpgradeScheduleTimeIsValid(updateTime string) bool {
+	// parse time
```

Comment:
I would change the description to something useful like how the time should look like

Reply:
I added an example!